relayer-engine/src/storage/storage.test.ts

```diff
@@ -42,7 +42,7 @@ describe("Storage tests", () => {
   beforeAll(async () => {
     store = await RedisWrapper.fromConfig({
       host: "localhost",
-      port: 6301,
+      port: 6379,
```

Comment:
I'd like to keep these tests using a non-standard port so that we don't accidentally wipe a real/prod redis.

Reply:
oops sorry, this was an accident.